source/adapter/scylladb/initializedb/db.go

```diff
@@ -0,0 +1,78 @@
+/*
+Package initializedb provides functionality for initializing a connection to ScyllaDB database using to gocql library.
+
+Note: Make sure to handle errors appropriately when using this package.
+*/
+package initializedb
+
+import (
+	"fmt"
+	"github.com/gocql/gocql"
+	"github.com/ormushq/ormus/source/adapter/scylladb"
+	"log"
+	"time"
+)
+
+type ScyllaDBConnection struct {
+	consistency gocql.Consistency
+	keyspace    string
+	hosts       []string
+}
+
+const (
+	timeoutCluster = 5 * time.Second
+
+	// numRetries represents the number of retries in the ExponentialBackoffRetryPolicy.
+	numRetries = 5
+
+	// minRetryDelay represents the minimum delay duration in the ExponentialBackoffRetryPolicy.
+	minRetryDelay = time.Second
+
+	// maxRetryDelay represents the maximum delay duration in the ExponentialBackoffRetryPolicy.
+	maxRetryDelay = 10 * time.Second
+)
+
+/*
+The 'createCluster' method creates and returns a gocql.ClusterConfig
+based on the provided connection parameters. It also sets additional configurations,
+such as timeout and retry policies.
+*/
+func (conn *ScyllaDBConnection) createCluster() *gocql.ClusterConfig {
+	cluster := gocql.NewCluster(conn.hosts...)
+	cluster.Consistency = conn.consistency
+	cluster.Keyspace = conn.keyspace
+	cluster.Timeout = timeoutCluster
+	cluster.RetryPolicy = &gocql.ExponentialBackoffRetryPolicy{
+		NumRetries: numRetries,
+		Min:        minRetryDelay,
+		Max:        maxRetryDelay,
+	}
+	cluster.PoolConfig.HostSelectionPolicy = gocql.TokenAwareHostPolicy(gocql.RoundRobinHostPolicy())
+	log.Println("cluster was created.")
```

Comment:
Please remove redundant logs or at least use slog.Debug (the logger package in ormus codebase).